lib/libutils/ext/include/trace.h

```diff
@@ -14,6 +14,15 @@
 #define MAX_PRINT_SIZE      256
 #define MAX_FUNC_PRINT_SIZE 32
 
+#define COLOR_NORMAL	"\x1B[0m"
+#define COLOR_RED	"\x1B[31m"
+#define COLOR_GREEN	"\x1B[32m"
+#define COLOR_YELLOW	"\x1B[33m"
+#define COLOR_BLUE	"\x1B[34m"
+#define COLOR_MAGENTA	"\x1B[35m"
+#define COLOR_CYAN	"\x1B[36m"
+#define COLOR_WHITE	"\x1B[37m"
```

Comment:
Could you add a specific commit in your series and move this change into?
```
libutils: trace: add colorization escape codes

Adds macros in trace.h for colorization of console trace messages using
ECMA-48 [1] and ISO/IEC 6429:1992 [2] escape code values.

Link: [1] https://www.ecma-international.org/publications-and-standards/standards/ecma-48/
Link: [2] https://www.iso.org/standard/12782.html
Signed-off-by: ...
```

Feel free to rephrase the comit message as you like. Note i've pointed the 2 specs as ECMA-48 spec is free to download whereas ISO/IEC asks for a fee. Or maybe the wikipedia link (https://en.wikipedia.org/wiki/ANSI_escape_code) is enough.

Reply:
Done